notificationdata.cpp

```diff
@@ -23,6 +23,15 @@ QString NotificationData::text() const {
 }
 
 QString NotificationData::img() const {
+    const QString imageSrc = imgSrc();
+    if(imageSrc != "")
+      return imageSrc;
+    else
+      return this->getDefaultImage(_type);
```

Comment:
Do not use **else** if your user return on the true block 
```suggestion

    return this->getDefaultImage(_type);
```